db/migrate/20230529094940_delete_admin_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteAdminFromUsers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :admin
```

Comment:
```suggestion
    remove_column :users, :admin, default: false
```

pour que celle-ci soit reversible (à tester btw)

Reply:
Je suis parti (avec erreur) du principe que Rails me laisserais pas faire lol

La bonne réponse était en fait:

```
remove_column :users, :admin, :boolean, default: false
```